libsplinter/src/admin/store/diesel/schema.rs

```diff
@@ -132,3 +132,20 @@ table! {
         endpoint -> Text,
     }
 }
+
+allow_tables_to_appear_in_same_query!(
```

Comment:
Are the proposal tables used in any of the same queries as the circuit tables?

Reply:
No, I can separate them